docs/content/custom-resources/_index.md

```diff
@@ -188,73 +131,57 @@ spec:
 
-A frequent use case is to create a PostgreSQL cluster with S3 or a S3-like
-storage system for storing backups. This requires adding a Secret that contains
-the S3 key and key secret for your account, and adding some additional
-information into the custom resource.
-
-#### Step 1: Create the pgBackRest S3 Secrets
+As part of this process, the PostgreSQL Operator creates several Secrets that
+contain the credentials for three user accounts that must be present in order
+to bootstrap a PostgreSQL cluster. These are:
 
-As mentioned above, it is necessary to create a Secret containing the S3 key and
-key secret that will allow a user to create backups in S3.
+- A PostgreSQL superuser
+- A replication user
+- A standard PostgreSQL user
 
-The below code will help you set up this Secret.
+The Secrets represent the following PostgreSQL users and can be identified using
+the below patterns:
 
-```
-# this variable is the name of the cluster being created
-pgo_cluster_name=hippo
-# this variable is the namespace the cluster is being deployed into
-cluster_namespace=pgo
-# the following variables are your S3 key and key secret
-backrest_s3_key=yours3key
-backrest_s3_key_secret=yours3keysecret
+| PostgreSQL User | Type        | Secret Pattern                     | Notes |
+| --------------- | ----------- | ---------------------------------- | ----- |
+| `postgres`      | Superuser   | `<clusterName>-postgres-secret`    | This is the PostgreSQL superuser account. Using the above example, the name of the secret would be `hippo-postgres-secret`. |
+| `primaryuser`   | Replication | `<clusterName>-primaryuser-secret` | This is for the managed replication user account for maintaining high availability. This account does not need to be accessed. Using the above example, the name of the secret would be `hippo-primaryuser-secret`. |
+| User            | User        | `<clusterName>-<User>-secret`      | This is an unprivileged user that should be used for most operations. This secret is set by the `user` attribute in the custom resources. In the above example, the name of this user is `hippo`, which would make the Secret `hippo-hippo-secret` |
 
-kubectl -n "${cluster_namespace}" create secret generic "${pgo_cluster_name}-backrest-repo-config" \
-  --from-literal="aws-s3-key=${backrest_s3_key}" \
-  --from-literal="aws-s3-key-secret=${backrest_s3_key_secret}"
+To extract the user credentials so you can log into the database, you can use
+the following JSONPath expression:
 
-unset backrest_s3_key
-unset backrest_s3_key_secret
 ```
+# namespace that the cluster is running in
+export cluster_namespace=pgo
+# name of the cluster
+export pgo_cluster_name=hippo
+# name of the user whose password we want to get
+export pgo_cluster_username=hippo
 
-#### Step 2: Creating the PostgreSQL User Secrets
+kubectl -n "${cluster_namespace}" get secrets \
+  "${pgo_cluster_name}-${pgo_cluster_username}-secret" -o "jsonpath={.data['password']}" | base64 -d
```

Comment:
📝 `kubectl` can do the decoding from the golang template format: 

```
-o 'go-template={{ index .data "password" | base64decode }}'
```

Reply:
I think I got that reference from official docs on `kubectl`. Noted for the future.